src/theme/components/tabs.ts

```diff
@@ -0,0 +1,51 @@
+const CustomTabs = {
+  baseStyle: {
+    tablist: {
+      borderColor: "ui.black",
+      maxWidth: {
+        base: "70%",
+        lg: "100%",
+      },
+    },
+    tab: {
+      color: "black !important",
+      paddingInlineStart: 4,
+      paddingLeft: 4,
+      border: "none",
+      borderColor: "ui.black",
+      paddingRight: {
```

Comment:
As per the UX spec: 1em padding left and right on mobile or "base" and then 3em on desktop or "md" (600px).